cmake/modules/Findthrift.cmake

```diff
@@ -17,14 +17,11 @@ find_library(
   HINTS ${thrift_HOME} ENV thrift_HOME /usr/local /opt/local
   PATH_SUFFIXES lib lib64)
 
-if(thrift_FOUND AND NOT (TARGET thrift::libthrift))
-  add_library(thrift::libthrift UNKNOWN IMPORTED)
-
-  set_target_properties(
-    thrift::libthrift
-    PROPERTIES IMPORTED_LOCATION ${thrift_LIBRARIES}
-               INTERFACE_INCLUDE_DIRECTORIES ${thrift_INCLUDE_DIR})
-endif()
+add_library(thrift::libthrift UNKNOWN IMPORTED)
+set_target_properties(
+  thrift::libthrift
+  PROPERTIES IMPORTED_LOCATION ${thrift_LIBRARIES}
+	     INTERFACE_INCLUDE_DIRECTORIES ${thrift_INCLUDE_DIR})
```

Comment:
you might want to transpose the `if(thrift_FOUND..)` block with the `find_package_handle_standard_args(thrift...)` statement, which sets `thrift_FOUND`.

Reply:
@tchaikov thanks, locally seems to do the trick, will test